chapters/annotations.tex

```diff
@@ -188,10 +188,19 @@ \subsubsection{Plot Curves}\label{plot-curves}
 end Curve;
 \end{lstlisting}
 
-The mandatory \lstinline!x! and \lstinline!y! expressions are restricted to be component references referring to a scalar variable or \lstinline!time!.
-It is an error if \lstinline!x! or \lstinline!y! does not designate a scalar variable.
+The mandatory \lstinline!x! and \lstinline!y! expressions are restricted to be result references in the form of \lstinline[language=grammar]!result-reference! in the grammar (\cref{expressions1}), referring to a scalar variable (or a derivative thereof) or \lstinline!time!.
+It is an error if \lstinline!x! or \lstinline!y! does not designate a scalar result.
+If \lstinline!x! or \lstinline!y! is a derivative, \lstinline!der(v, $n$)!, then $n$ must not exceed the maximum amount of differentiation applied to \lstinline!v! in the model.
 When the \lstinline!unit! of an \lstinline!Axis! is non-empty, it is an error if the unit of the corresponding \lstinline!x! or \lstinline!y! expression (i.e., a variable's \lstinline!unit!, or second for \lstinline!time!) is incompatible with the axis unit.
 
+The set of available result variable and derivative trajectories in a simulation result is tool-specific, but it is recommended to include at least the ones being used in figures.
+It is not standardized how a tool should handle missing results for valid result references.
+
+\begin{nonnormative}
+While the syntax for referring to a second order derivative is \lstinline!der(v, 2)!, the appearance is left for tools to decide.
+For example, a tool might choose to present this as \lstinline!der(der(v))! or even $v''$.
```

Comment:
Personally I would prefer `\ddot{v}` over `$v''$` to denote the time-derivative, although it is less important as it is non-normative. See https://en.wikipedia.org/wiki/Notation_for_differentiation
```suggestion
For example, a tool might choose to present this as \lstinline!der(der(v))! or even $\ddot{v}$.
```

Reply:
This looks really a matter of taste. BTW I'm having endless discussions with my friends from thermodynamics, who insist on using $\dot{m}$ for flow rate and $\dot{Q}$ for thermal power. It's hopeless

Reply:
To me dot is for time-derivatives, and prime for more general derivatives.
(And flow rate and thermal power aren't really derivatives.)

Reply:
To me too. But I couldn't convince them. And they also use Modelica. Sometimes life's complicated...

Reply:
Personally, I like to reserve `$\dot{x}$` for something like a function formal parameter meant for evaluation at `$x'(t)$`, where `$x'$` is the derivative of `$x$`.  In the context of simulation results, each notation is meaningful:
- `$x'$` corresponds more closely to `der(x)` being thought of as the differentiation operator applied to `x`.
- `$\dot{x}$` would emphasis that this is should not be seen as (plotting) an expression of `x`, but as (plotting) the independent result "variable" `$\dot{x}$`.

While the interpretations above actually speak in favor of `$\dot{x}$`, it was a mistake from the beginning to use math style for the variable `v`.  Presenting variable names in math style seems like a stylistic mess to me, so it would have been better to have this discussion around `$\text{\lstinline{v}}'$`.  If we then think of a longer variable name, such as `aimcM.rotorCage.electroMagneticConverter.singlePhaseElectroMagneticConverter[1].Phi.re`, then I think it becomes clear that the `$\dot{…}$ approach has problems.

Can't we just drop the $v''$ variant from the text?

Reply:
Yes, I agree that dropping `$v''$` is probably best.